src/nvim/testdir/test_timers.vim

```diff
@@ -47,6 +47,9 @@ func Test_repeat_many()
   call timer_stopall()
   let g:val = 0
   let timer = timer_start(50, 'MyHandler', {'repeat': -1})
+  if has('mac')
+    sleep 200m
```

Comment:
For future reference, there's now `LoadAdjust()` function ( `load_adjust` in Lua). 0afd452ef19ed08db8e74d72c0316bc40ac9c9bd